src/main/preload/mattermost.js

```diff
@@ -373,3 +374,25 @@ ipcRenderer.on(CALLS_JOIN_REQUEST, (event, message) => {
 window.addEventListener('resize', () => {
     ipcRenderer.send(VIEW_FINISHED_RESIZING);
 });
+
+let isPasswordBox = false;
```

Comment:
Are we positive any focus out event always come before a new focusin?

Reply:
From testing it would appear so, I'm not sure if by definition it has to.

Either way it's low risk if not, in the worst case the secure input stays on a little longer.